components/Modals/SignUp.tsx

```diff
@@ -58,21 +59,20 @@ export default function SignUpModal({ open, onClose, onConfirm }) {
 
         <h2 className="mb-4 text-xl font-bold">Choose an avatar</h2>
 
-        <div className="mb-5 flex cursor-pointer flex-wrap justify-between">
+        <Slider>
```

Comment:
Let's rename this to ` HorizontalScrollView` or something akin to that. I'd reserve Slider for [these](https://www.w3schools.com/howto/howto_js_rangeslider.asp).

Reply:
Sure!

Addressed in 339d190d6fe8f5a054e46f8e988d834c63f40c4e